src/modules/Addscores.js

```diff
@@ -0,0 +1,14 @@
+const addScore = async (url, name, score) => {
+  await fetch(url, {
+    method: 'POST',
+    body: JSON.stringify({
+      user: name,
+      score,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  }).then((response) => response.json());
+};
```

Comment:
- [ ] Let's try to understand some requirements for this project.
![game](https://github.com/ANTOINE1128/Leaderboard/assets/53124548/f2fe8f7c-5db7-4a3c-ab3e-d324d1ed0ebe)

These two requirements mean that we should send a post request (empty), and its return should give us an ID which we should then use as a key. Remember we just want to get only the scores for this game, but your app returns different data which we do not want.